e3nn/o3/_spherical_harmonics.py

```diff
@@ -84,12 +85,20 @@ def __init__(
                 f"spherical_harmonics maximum l implemented is {_lmax}, send us an email to ask for more"
             )
 
+        # TODO: Using this to turn off the torch.jit.script for torch.compile.
+        # Please remove this when torch.script is depracated
+        if get_optimization_defaults()["jit_script_fx"]:
+            self.sph_func = torch.jit.script(_spherical_harmonics)
+        else:
+            # Turn off for torch.compile
+            self.sph_func = _spherical_harmonics
```

Comment:
I am not sure if this will be picklable since it's a function and `ScriptFunction` may not be picklable, is that included in the tests for sh?

Reply:
```jit_script_fx=True``` does seem to be unpickable so open to any suggestions that you might have 
https://github.com/pytorch/pytorch/issues/61210